### PR TITLE
Cu 86b17gex4 hosted media thumbnails for team maverick project ethan freire

### PR DESCRIPTION
**Pull request by ethanfreire**

### CODE REVIEW

**Comment by allantito** [comment]
Task linked: [CU-86b17gex4 Hosted media thumbnails for team maverick project](https://app.clickup.com/t/86b17gex4)

File: P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx [hunk 1]
```diff
@@ -1,59 +1,50 @@
...
-                                    <div className="media-title">{movie.title}</div>
-                                </div>
-                            </div>
-                        )
-                    :
-
-                        (
-                            <div>
-                                <img src={imgSrc} alt={movie.title} />
+                imgSrc ? (
```
Comment (ethanfreire) [inline P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx:17]: This large ternary operator is hard to read. I suggest refactoring this code and decomposing the logic.

File: P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx [hunk 2]
```diff
@@ -1,59 +1,50 @@
...
+                    type.toLowerCase() === "trending" ? (
+                        <div style={{ borderRadius: "20px", width: "540px", height: "280px", backgroundImage: `url(${imgSrc})`, backgroundPosition: "center", backgroundRepeat: "no-repeat", backgroundSize: "cover" }}>
```
Comment (ryanRATM) [inline P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx:19]: if going to have multiple ways to render a movie card like regular/trending you could make them their own component.

Comment (ethanfreire) [reply P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx:19]: Agreed this code should be refactored.

**Pull request merged**